pkg/ast/ast.go

```diff
@@ -1263,7 +1263,7 @@ type CreateTable struct {
 	Name              *Ident
 	Columns           []*ColumnDef
 	PrimaryKeys       []*IndexKey
-	ForeignKeys       []*ForeignKey
+	TableConstraints  []*TableConstraint       // optional
```

Comment:
I'm not sure this `// optional` comment is necessary. Contrary, there is `// len(Whens) > 0` comment, so we assumed all slice fields without comment contain an empty slice.

Reply:
OK, makes sense.